### ISSUE TITLE
Two output node should give error

### ISSUE THREAD

**Issue by srivastava9** [scorelab/TensorMap#114]
While making a neural network architecture, It should give an error when one tries to make two output node.
 
![Screenshot from 2020-03-14 21-44-53](https://user-images.githubusercontent.com/45288400/76686068-c22dac00-663e-11ea-8013-45aafb3b4cc7.png)

Right now it does not give any error.

@orionpax00 Need your update on this one also :)

**Comment by vasudev13** [comment]
@srivastava9 , I think its okay to have multiple input and output nodes(as mentioned in #115). Francois Chollet, who is the creator of Keras, writes about them in his book Deep Learning with Python (Specifically Chapter 7: Advanced deep-learning best practices).

**Comment by srivastava9** [comment]
True, But I guess we are not making such advance model right now,Anyway when it goes to backend it only accepts one input and output model right now.Therefore I opened this issue.
@ivantha @orionpax00 need your suggestion :)
